Source/WebCore/platform/graphics/ca/GraphicsLayerCA.h

```diff
@@ -153,6 +153,7 @@ class GraphicsLayerCA : public GraphicsLayer, public PlatformCALayerClient {
     WEBCORE_EXPORT void setContentsToPlatformLayerHost(LayerHostingContextIdentifier) override;
     WEBCORE_EXPORT void setContentsToVideoElement(HTMLVideoElement&, ContentsLayerPurpose) override;
     WEBCORE_EXPORT void setContentsDisplayDelegate(RefPtr<GraphicsLayerContentsDisplayDelegate>&&, ContentsLayerPurpose) override;
+    WEBCORE_EXPORT PlatformLayerIdentifier setContentsToAsyncDisplayDelegate(RefPtr<GraphicsLayerContentsDisplayDelegate>, ContentsLayerPurpose);
```

Comment:
Should the first arg be a `RefPtr<>&&` ?

Reply:
I don't think so, we want to hold a new reference to the delegate (since the caller wants to still have a reference too).